trx/io.py

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import logging
+import tempfile
+
+try:
+    from dipy.io.stateful_tractogram import StatefulTractogram
+    from dipy.io.streamline import load_tractogram, save_tractogram
+    dipy_available = True
+except ImportError:
+    dipy_available = False
+
+from trx.utils import split_name_with_gz
+
+
+def get_trx_tmpdir():
+    if os.getenv('TRX_TMPDIR') is not None:
+        if os.getenv('TRX_TMPDIR') == 'use_working_dir':
```

Comment:
@frheault I gather this would be equivalent to `TRX_TMPDIR=.` ?  Is `.` also valid in Windows?

Reply:
It is equivalent, but I wanted to write it explicitly rather than using only `.`
I have not tried it on windows.

Reply:
Sounds good!